test/format/bulk.c

```diff
@@ -186,20 +190,24 @@ table_load(TABLE *base, TABLE *table)
             break;
         }
 
+        report_progress = (keyno < 5000 && keyno % 10 == 0) || keyno % 5000 == 0;
+        /* Report on progress. */
+        if (report_progress)
+            track(track_buf, keyno);
+
         /*
          * When first starting up, restart the enclosing transaction every 10 operations so we never
          * end up with an empty object. After 5K records, restart the transaction every 5K records
          * so we don't overflow the cache.
+         *
+         * If we are loading a mirrored table, commit after every operation to ensure that we are
```

Comment:
`commit after every operation` -> `commit periodically` sounds better IMO, after every operation sounds like we are committing every single one.